### PR TITLE
LP-2243 Write unit tests for fetch_course_structures philu_command

### PR DESCRIPTION
**Pull request by muak50**
### Description

[LP-2243](https://philanthropyu.atlassian.net/browse/LP-2243)

### CODE REVIEW

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by ahmed-faraz46** [comment]
jenkins run all

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Comment by ahmed-faraz46** [comment]
jenkins run all

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by ahmed-faraz46 — APPROVED** [review]
LGTM 👍

File: common/djangoapps/philu_commands/management/commands/tests/test_fetch_course_structures.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+"""
+import mock
+from django.core.management import call_command
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class TestFetchCourseStructure(ModuleStoreTestCase):
+    @mock.patch('philu_commands.management.commands.fetch_course_structures.generate_course_structure')
```
Comment (zahid-arbisoft) [inline common/djangoapps/philu_commands/management/commands/tests/test_fetch_course_structures.py:12]: Please add class doc string

File: common/djangoapps/philu_commands/management/commands/tests/test_fetch_course_structures.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Tests for 'fetch_course_structures' command
+"""
+import mock
+from django.core.management import call_command
+
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.tests.factories import CourseFactory
+
+
+class TestFetchCourseStructure(ModuleStoreTestCase):
```
Comment (zahid-arbisoft) [inline common/djangoapps/philu_commands/management/commands/tests/test_fetch_course_structures.py:11]: Can we write one more test with invalid course id and assert that `generate_course_structure` is not called

`assert not generate_course_structure.called`

**Review by zahid-arbisoft — CHANGES_REQUESTED** [review]

**Comment by philanthropyu** [comment]
Your PR has finished running tests. The following contexts failed:
* jenkins/python

**Review by zahid-arbisoft — APPROVED** [review]

**Pull request merged**